### DISCUSSION TITLE
iOS version has a transparent statusbar

### DISCUSSION THREAD

**Discussion by jankal13** [quasarframework/quasar#9503] [CLI - Capacitor / Cordova modes]
Hello together, 

I have a Quasar App which seems to have some issues with capacitor/iOS. When I run the iOS version it seems that the statusbar of the app becomes transparent/translucent although I defined it in the plist as "lightcontent/white". 
Also it does not matter if the phone has a notch or not. The statusbar always stays transparent. I just can get rid of it if I use "viewport-fit=cover" which I don't want to use as it screws up my app's layout.

Does anybody know why the statusbar is transparent, how I can colour it (e.g. make it white) and what I missed?

Thank you and best regards
JK

<img width="377" alt="Screenshot 2021-05-31 at 23 07 31" src="https://user-images.githubusercontent.com/15088395/120245959-bee40780-c266-11eb-8c70-069ff7bc45e9.png">

<img width="370" alt="Screenshot 2021-05-31 at 23 10 00" src="https://user-images.githubusercontent.com/15088395/120245859-7cbac600-c266-11eb-9f1e-a4cc2bba41e9.png">

**Comment by ontwikkelfabriek** [comment]
@mrzyte 

Did you use the Capacitor status bar plugin?
https://capacitorjs.com/docs/apis/status-bar

Here's the Cordova plugin. I know that one works to create solid non transparent status bars in iOS.
https://cordova.apache.org/docs/en/10.x/reference/cordova-plugin-statusbar/

**Reply by jankal13** [reply]
Thank you for your answer. I tried the suggested cordova plugin but it still results in a transparent status bar. I guess there is something wrong with the iOS settings in general :( Have you set up some specific settings in your iOS project to make it non transparent?

**Reply by ontwikkelfabriek** [reply]
@mrzyte 

this is my config.xml settings for statusbar inside /src-cordova
```
<preference name="StatusBarStyle" value="lightcontent" />
<preference name="StatusBarOverlaysWebView" value="false" />
<preference name="StatusBarBackgroundColor" value="#F79432" />
```

and the viewport quirks for IOS:
https://cordova.apache.org/docs/en/10.x/reference/cordova-plugin-statusbar/#ios-quirks
